pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -85,7 +125,26 @@ WCAG2ICT follows the [W3C](https://www.w3.org) format for technical reports, whi
 
 ## Who develops WCAG2ICT {#wg}
 
-WCAG2ICT is developed by the [WCAG2ICT Task Force](https://www.w3.org/WAI/GL/WCAG2ICT-TF/), provided for public review, and approved by the Accessibility Guidelines Working Group ([AG WG](https://www.w3.org/WAI/GL/)), which is part of the World Wide Web Consortium ([W3C](https://www.w3.org/)) Web Accessibility Initiative ([WAI](https://www.w3.org/WAI/)).
+The WCAG2ICT Group Note is developed by a [Task Force (TF)](https://www.w3.org/WAI/about/groups/task-forces/wcag2ict/) of the [Accessibility Guidelines Working Group (AG WG)](https://www.w3.org/WAI/about/groups/agwg/). The [work statement](https://www.w3.org/WAI/about/groups/task-forces/wcag2ict/work-statement/) defines the objectives, scope, and approach for the WCAG2ICT Group Note. The participants of the WCAG2ICT TF are volunteers drawn from [W3C member organizations](https://www.w3.org/membership/list/) and [invited experts](https://www.w3.org/invited-experts/) and are supported by [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/about/#wai-staff).
+
+### Related Documents
```

Comment:
Traditionally, we have avoided having all these pointers to external resources on the overview pages. I think these belong more on the Note itself. I presume this is one thing that other WAI Staff may have comments on. Would we be OK moving these to the document itself (some of them are already there) and pointing to that section if we want to explicitly call them out in here.